src/playlist/playlistview.cpp

```diff
@@ -214,6 +214,9 @@ PlaylistView::PlaylistView(QWidget *parent)
 
   dynamic_controls_->hide();
 
+  // To proper scale all pixmaps
+  device_pixel_ratio = this->devicePixelRatioF();
```

Comment:
Can we omit this-> here? Can we call devicePixelRatioF() directly on the functions?

Reply:
Sure, I will do a commit to reflect that and the variable above.